src/uct/ib/base/ib_iface.c

```diff
@@ -1760,6 +1760,10 @@ uct_ib_iface_estimate_perf(uct_iface_h iface, uct_perf_attr_t *perf_attr)
         perf_attr->latency = iface_attr.latency;
     }
 
+    if (perf_attr->field_mask & UCT_PERF_ATTR_FIELD_MAX_INFLIGHT_EPS) {
+        perf_attr->max_inflight_eps = SIZE_MAX;
```

Comment:
we need to set a specific value for RC as well

Reply:
I just got "timeout waiting for replies" - so, it seems that it doesn't help

Reply:
anyway, I'd update RC in the separate PR to unlock working on https://github.com/openucx/ucx/pull/7899 PR

Reply:
i think this change is supposed to be here as well. This PR defines the API and also sets its value for all the transports. Why need to skip one of them?

Reply:
@yosefe, wdyt?

Reply:
sure, let's add to this PR